deploy/helm/distributed-compute-operator/templates/hooks.yaml

```diff
@@ -0,0 +1,144 @@
+{{- if .Values.installCRDs }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "dco.serviceAccountName" . }}-hook
+  labels:
+    {{- include "dco.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade,post-delete
+    "helm.sh/hook-weight": "-3"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dominodatalab:operator:distributed-compute-operator
```

Comment:
ideally, these global names have some sort of release/ns identifier since they can conflict (though maybe that's just not a thing given this operator needs other global resources)

Reply:
ack. I think what I want to do after I merge this PR is create another strictly for all of the Helm chart changes; this is more of a _template_ for the final version. I can explore using Bitnami's common chart library and address all of your other chart concerns at that time.